includes/Settings.php

```diff
@@ -287,4 +288,15 @@ private function doIterate( $key, $options ) {
 		throw new SettingNotFoundException( "'{$key}' is not a valid settings key" );
 	}
 
+	private static function initLegacyMapping( &$configuration ) {
+
+		// When adding a deprecated mapping, extend the DeprecationNoticeTaskHandler
+		// with the corresponding entry
+
+		// Deprecated in 2.5, to be removed with 3.1
```

Comment:
> Removal in a feature release? I suppose this is not truly a user-space thing... but still...

Doing an extra release just to remove something ... 3.0 is big so watch out for changes.